### ISSUE TITLE
Reactivating nightly

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-core#104]
We need to reactivate nightly, so that the latest commit of the branch nightly are always copied there.

I put this as urgent as we need it for all the other.

@giovanisp do you confirm that this is just on the nightly branch, right?
(assigned to you for control, and then to @polites75 )

CC @derekpa . So once a fix is done it needs to be merged with nightly to check it.

(a less urgent issue will be to estabilish a CI/CD system)

**Comment by giovanisp** [comment]
We have an issue about CI here #28  @coorasse take a look if you want to add something ;-)

@derekpa we need to deploy each commit or merge in nightly branch (tipically we'll want to merge into nightly once a day) directly where https://nightly.vilfredo.org is. Both core and client components should be automated. See here, instead of eroku, we'll use our servers. https://developer.github.com/guides/automating-deployments-to-integrators/

**Comment by giovanisp** [comment]
@derekpa, @pietrosperoni  news on this work labeled urgent ?

**Comment by pietrosperoni** [comment]
We are waiting for @polites75 to finish the new program to do it
